### ISSUE TITLE
reminder for taking survey

### ISSUE THREAD

**Issue by elvandy** [jefferythewind/appetite_tamer#10]
The reminder for the game works properly. However, the reminder for taking the survey was not working properly. Can Tim please check how we setup for that in the script?

**Comment by jefferythewind** [comment]
Hi @elvandy sure I will take a look.  How is the reminder supposed to work?

**Comment by jefferythewind** [comment]
@elvandy I just was prompted to play the game, then it was only about 15 minutes later I was sent a notification to take the survey...

**Comment by elvandy** [comment]
@jefferythewind Are you in the public mode or in the study mode? I know that it works in the study mode (when you sign in with activating the study code). However, I don't think that it works in the public mode. I haven't received any reminder when I am in the public mode. I will keep testing. Please let me know if it is confusing.

**Comment by jefferythewind** [comment]
yes i have study participant enabled.  I think it maybe like that by default?  A few other testers have responded that YES they are getting notifications about survey, and there is a bug that the survey can't finish.  See this: https://github.com/jefferythewind/appetite_tamer/issues/4

**Comment by elvandy** [comment]
@jefferythewind  I think that we don't want study participant enabled. We want to test the public mode at this moment. The study mode is too intense for the general public. One idea is to have someone in the study mode and someone in the public mode. The other idea is to have everyone in the study mode for a week and switch to the public mode for another week. In this way, we can make sure both modes work.

**Comment by jefferythewind** [comment]
@elvandy OK Andy, so I will need some specs about how the different modes are supposed to work, like exactly **when** they are supposed to receive **which** kind of reminder for each mode.  

Right now I'm assuming there are only 2 types of reminders 1) to play the game, 2) to take the survey.  If that is correct then it is just a case of setting the schedule for each type of mode (study participant or no).

**Comment by jefferythewind** [comment]
Hi Andy @elvandy, I found some notes in the code about how the notifications are designed, let me know if this is correct.  You'll see if that non-study people only get notifications every 2 days or so, so that may be why you didn't see any notifications come through.   I was in the study, and I **do not recall** receiving the notification 3 times each day.  Anyways we can work on the following rules and then I can update the code to accurately do them.

> //adds a notification to play the game.
>     //  Players in the study get notifs 3 times a day if they miss the interval they were supposed to play during.
>     //  Players not in the study get reminded 48hours after they last played.


> //Adds a notif to take the survey late at night for users in study


> //Add notif to take the Survey
>     // Players in study get asked 3 times a day. either 5min or 15min after.
>     // The reason it is 5min instead of right after is bc players in study will play multiple games
>     // Players not is study get asked 48hrs after they play randomly
